cmd/amp/commands.go

```diff
@@ -128,7 +129,7 @@ func addCommands(cmd *cobra.Command, c cli.Interface) {
 
 func info(c cli.Interface) error {
 	s := c.Server()
-	c.Console().Infof("[%s]\n", s)
+	fmt.Fprintf(c.Err(), "[%s]\n", s)
```

Comment:
For the output to be redirected to standard error to make the piped commands with -q to work